src/plugins/bfetch/public/streaming/from_streaming_xhr.ts

```diff
@@ -41,14 +45,27 @@ export const fromStreamingXhr = (
 
   xhr.onprogress = processBatch;
 
+  const onBatchAbort = () => {
+    if (xhr.readyState !== 4) {
```

Comment:
👍 good check, I guess this makes the HTTP abort a noop when it is aborted after all items succeed.